ngraph/frontend/frontend_manager/include/frontend_manager/frontend_manager.hpp

```diff
@@ -27,10 +27,10 @@ class FRONTEND_API FrontEndManager final {
     FrontEndManager();
 
     /// \brief Default move constructor
-    FrontEndManager(FrontEndManager&&);
+    FrontEndManager(FrontEndManager&&) noexcept;
```

Comment:
Why do you add `noexcept`?

Reply:
It is not related to PR, just a 'IDE warning' fix, it is good style to declare move constructor/operator= as noexcept (some algorithms have more efficient implementations if such constructors are noexcept)